css/style.css

```diff
@@ -27,12 +29,13 @@ Developer @GA in NYC
      height: 0;
 }
 
+
 .clear {
   clear: both;
 }
 
 .alignright {
-  float: right;
+   : right;
```

Comment:
There is missing a selector, so there should be either a "float" or "text-align" before the ':'.